src/components/Layout/UsersTable/UsersTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import MaterialTable from 'material-table'
+
+export default function MaterialTableDemo() {
```

Comment:
Change the name of this function to replicate the one of the component and also make it consisten with the rest of the components `const UsersTable: React.FC ...`